### PR TITLE
s3, curvefs client, metasever: add s3 log prefix & refresh inode only…

### LINKED ISSUE #977: Specify the S3 log directory in the client.conf
**Is your feature request related to a problem? (你需要的功能是否与某个问题有关?)**
https://github.com/opencurve/curve/issues/970
According to the curve-fuse log, it took 5 minutes to asynchronously upload data to S3. We would like to take a look at the S3 logs, why are they being processed so slowly. But S3 logs are not mapped to the physical machine directory, so you cannot see the previous logs. 

**Describe the solution you'd like (描述你期望的解决方法)**
We can configure the S3 log path in client.conf.

**Describe alternatives you've considered (描述你想到的折衷方案)**

**Additional context/screenshots (更多上下文/截图)**

### PR DESCRIPTION
**Pull request by xu-chaojie**
… refresh s3 info

### What problem does this PR solve?

Issue Number: close #977  

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?): 

### Check List

- [x] Relevant documentation/comments is changed or added
- [x] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

File: curve-ansible/roles/generate_config/defaults/main.yml [hunk 1]
```diff
@@ -250,6 +250,7 @@ s3_max_connections: 32
 s3_connect_timeout: 60000
 s3_request_timeout: 10000
 s3_loglevel: 4
+s3_logPrefix: ./
```
[the remark this answers is not in the document]
Comment (xu-chaojie) [reply curve-ansible/roles/generate_config/defaults/main.yml:253]: done

File: curvefs/conf/client.conf [hunk 2]
```diff
@@ -111,6 +111,7 @@ s3.connect_timeout=60000
 s3.request_timeout=10000
 # Off = 0,Fatal = 1,Error = 2,Warn = 3,Info = 4,Debug = 5,Trace = 6
 s3.loglevel=4
+s3.logPrefix=/data/logs/curvefs/aws_
```
[the remark this answers is not in the document]
Comment (xu-chaojie) [reply curvefs/conf/client.conf:114]: done

File: conf/s3.conf [hunk 3]
```diff
@@ -18,6 +18,7 @@ s3.connect_timeout=60000
 s3.request_timeout=10000
 # Off = 0,Fatal = 1,Error = 2,Warn = 3,Info = 4,Debug = 5,Trace = 6
 s3.loglevel=4
+s3.logPrefix=./aws_
```
[the remark this answers is not in the document]
Comment (xu-chaojie) [reply conf/s3.conf:21]: done

File: curvefs/src/client/rpcclient/metaserver_client.cpp [hunk 4]
```diff
@@ -516,12 +517,11 @@ MetaStatusCode MetaServerClientImpl::GetOrModifyS3ChunkInfo(
...
             return ret;
-        } else if (response.has_appliedindex() &&
-            (!returnInode || response.has_inode())) {
+        } else if (response.has_appliedindex()) {
             metaCache_->UpdateApplyIndex(CopysetGroupID(poolID, copysetID),
                                          response.appliedindex());
-            if (returnInode) {
-                out->CopyFrom(response.inode());
+            if (returnS3ChunkInfoMap) {
+                out->swap(*response.mutable_s3chunkinfomap());
```
Comment (wu-hanqing) [inline curvefs/src/client/rpcclient/metaserver_client.cpp:524]: you should check whether `out` is nullptr or not

**Review by ilixiaocui — APPROVED** [review]

File: curvefs/src/client/rpcclient/metaserver_client.cpp (hunk 4 above)
Comment (xu-chaojie) [reply curvefs/src/client/rpcclient/metaserver_client.cpp:525]: done

**Review by wu-hanqing — APPROVED** [review]

**Pull request merged**